src/visualizers/Differences.ts

```diff
@@ -38,6 +38,28 @@ than the number of elements in the sequence')_
             if (l < 1n) stat.addError('Need at least one row')
         },
     },
+    /** md
+- **Digits displayed:** The maximum number of digits to display for each
+  sequence entry and difference displayed. If a number to be shown has more
```

Comment:
I found that hard to parse.  How about "The maximum number of digits to show for each number displayed."